CONTRIBUTING.md

```diff
@@ -122,6 +122,48 @@ branch. Steps:
    git push -u origin my-feature
    ```
 
+## Local Developer Setup
+1. Make sure you have a compatible version of Python 3 installed
+2. From the project root, create a virtual environment and install all development dependencies
+
+   **Bash**
+    ```bash
+   # Create virtual environment
+   python3.8 -m venv venv
```

Comment:
is there a reason for specifying 3.8 (here and elsewhere)?

Reply:
I specified 3.8 because it is a version supported by graspologic.  I will add a comment saying that you can substitute any supported version.  If you have multiple versions installed it helps to specify which one you want.